packages.dhall

```diff
@@ -1,5 +1,5 @@
 let upstream =
-      https://github.com/purescript/package-sets/releases/download/psc-0.13.8-20200831/packages.dhall sha256:cdb3529cac2cd8dd780f07c80fd907d5faceae7decfcaa11a12037df68812c83
+      https://github.com/purescript/package-sets/releases/download/psc-0.14.2-20210713/packages.dhall sha256:654c3148cb995f642c73b4508d987d9896e2ad3ea1d325a1e826c034c0d3cd7b
```

Comment:
I realized halfway through that this is actually slightly ahead of TPS package set (https://github.com/purescript/trypurescript/blob/ee9ca742c6c34901303043f64815c9794d23cd4c/staging/packages.dhall#L2) I don't know that that would lead to any incompatibility

Reply:
It shouldn't, but thanks for notifying us of that.

Reply:
Also, a `v0.14.3` package set was just released. Could you verify whether or not using that package set works?

Reply:
4848fb1